builder-base/install.sh

```diff
@@ -119,6 +121,21 @@ yum install -y \
     vim \
     which
 
+yum clean all
+rm -rf /var/cache/{amzn2extras,yum,ldconfig}
+find /var/log -type f | while read file; do echo -ne '' > $file; done
+go clean --modcache
+# pip cache
```

Comment:
This is too early for some of this to run. I think you can leave the yum and license stuff and move the go/pip clean up back towards the end.

Reply:
Done